### PR TITLE
Checking in code for database on private subnet of diff VPC

### PR DESCRIPTION
**Pull request by NatarajBTI**
@padmankosalaram @shajeena @amitmangalvedkar - Pls review the code changes for database on private subnet of different VPC. Also, let me know if I can remove docdb parameter in CFT file & docdb related VPC peering code because I feel Document DB related parameter field in CFT files & other code related to Document DB in our multicloud bootstrap code looks redundant & instead we can have single parameter field for database provisioned VPC ID (say DBProvisionedVPCId) & same code in our multicloud-bootstrap covering all databases such as DB2, Oracle, MSSQL, & Document DB.

### CODE REVIEW

File: aws/deploy.sh
Comment (shajeena) [inline aws/deploy.sh:1]: Line 311 ,should be invoked only for manage ? 
not required for MAS +Core... shouldnt we have a condition when manage=true sh $GIT_REPO_HOME/aws/db/db-create-vpc-peer.sh

File: aws/db/db-create-vpc-peer.sh
Comment (shajeena) [inline aws/db/db-create-vpc-peer.sh:1]: is this common code for documentDB and private code ?

File: pre-validate.sh
Comment (shajeena) [inline pre-validate.sh:1]: Its called in prevalidate and in deploy.sh  - T_REPO_HOME/aws/db/db-create-vpc-peer.sh -- for validation ?

File: aws/deploy.sh
Comment (NatarajBTI) [reply aws/deploy.sh:1]: Yes, this should be invoked only for manage & that's why I have included that check & only if DBProvisionedVPCId parameter value is entered by the user on CFT page then this code block executes. So this condition itself is enough & one more condition like when manage=true isn't required here.

**Review by shajeena — APPROVED** [review]

**Pull request merged**